src/handler/handle.go

```diff
@@ -2,9 +2,21 @@ package handler
 
 import (
 	"fmt"
+	"log"
 	"net/http"
+	"whiteboard/state"
+
+	"google.golang.org/protobuf/proto"
 )
 
-func HelloWorld(w http.ResponseWriter, req *http.Request) {
-	fmt.Fprintf(w, "Hello World")
+func GetUpdates(w http.ResponseWriter, req *http.Request) {
+	update := state.StateUpdate{
+		Version: "abra",
+		Data1:   1234,
+	}
+	out, err := proto.Marshal(&update)
+	if err != nil {
+		log.Printf("Marshaling error: ", err)
```

Comment:
I think it should be log.Fatal. This way we don't return 200 to the client with an empty body, but rather some meaningful error.